src/main/java/frc/util/Controllers/OI.java

```diff
@@ -0,0 +1,64 @@
+package frc.util.Controllers;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+
+/** 
+ * Handels xbox controllers.
+*/
+public class OI {
```

Comment:
we don't need an OI class, if we want to add one we should do so at a later point